docs/technologies.md

```diff
@@ -0,0 +1,423 @@
+# Technologies
+
+## Overview
+
+BigFlow provides support for the main big data data processing technologies on GCP:
+
+* [Dataflow](https://cloud.google.com/dataflow) (Apache Beam)
+* [BigQuery](https://cloud.google.com/bigquery)
+* [Dataproc](https://cloud.google.com/dataproc) (Apache Spark)
+
+However, **you are not limited** to these technologies. The only limitation is the Python language. What is more, you can
+mix all technologies in a single workflow.
+
+The provided utils allow you to build workflows easier and solve problems that must have been solved anyway.
```

Comment:
Maybe enumerate some of those problems here?

Reply:
done